Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
+
+image:
```

Comment:
I'm not sure I want to keep this task name: as we work with more open source tools as "first party" engines, it'll become more likely those other projects will already have makefiles, so I think "image" might be too generic. I'm wondering if we should standardize on something more explicit that we can expect to use without issue everywhere. Maybe "codeclimate_engine"? Thoughts on this @pbrisbin?